src/Tribe/Admin/Notice/Install_Event_Tickets.php

```diff
@@ -185,14 +188,74 @@ public function should_display_notice_activate(): bool {
 			return false;
 		}
 
+		$plugin_status_check    = $this->is_installed() && ! $this->is_active();
+		$is_admin_relevant_page = $this->is_tec_related_page();
+
 		/**
 		 * Filters whether the `Event Tickets` admin notice should display.
 		 *
 		 * @since 6.9.0
 		 *
 		 * @param bool $should_display True if the notice should display.
 		 */
-		return apply_filters( 'tec_events_admin_notice_event_tickets_should_display', $this->is_installed() && ! $this->is_active() && ! $this->is_install_plugin_page() );
+		return apply_filters( 'tec_events_admin_notice_event_tickets_should_display', $plugin_status_check && $is_admin_relevant_page );
+	}
+
+	/**
+	 * Checks if the current admin page is TEC related.
+	 *
+	 * @since TBD
+	 *
+	 * @return bool True if the current admin page is TEC related.
+	 */
+	public function is_tec_related_page(): bool {
+		// Not in the admin we don't even care.
+		if ( ! is_admin() ) {
+			return false;
+		}
+
+		global $current_screen;
+		
+		// No screen, bail.
+		if ( empty( $current_screen ) ) {
+			return false;
+		}
+		
+		// Match any TEC post type screens (events, organizers, venues).
+		$tec_post_types = [
+			Events_Main::POSTTYPE,
+			Events_Organizer::POSTTYPE,
+			Events_Venue::POSTTYPE,
+		];
+		
+		if ( in_array( $current_screen->post_type, $tec_post_types, true ) ) {
```

Comment:
@bordoni Did you happen to see [my earlier comment](https://github.com/the-events-calendar/the-events-calendar/pull/4994#discussion_r2003906547) about `in_array()`?

Reply:
Ahh I did I forgot to apply that